### DISCUSSION TITLE
Jams are liquids (to be drunk)?

### DISCUSSION THREAD

**Discussion by actual-nh** [CleverRaven/Cataclysm-DDA#49148] [Q&A] [answered]
In items/comestibles/fruit_dishes.json, jams are down as liquid (and as to be drunk), as is brandied fruit. What is the justification for this (or is it a bug)? EDIT: @Maleclypse answered on the discord (thanks!) - it's so they require a container. I will see about making that clear on the wiki.

**Comment by Maleclypse** [comment, marked as answer]
Answering here as well so it can be listed as answered.       Liquids and gasses require a container.  Other phases do not I believe.

**Reply by Maleclypse** [reply]
Sidenote sorry for selecting my response as the answer but this answers a question we'd had about how the Q&A category works.

**Reply by actual-nh** [reply]
No problem at all. All it takes is triage, actually.
